docs/unified/protectli/firmware-update.md

```diff
@@ -0,0 +1,143 @@
+# Firmware update
+
+The following documentation describes the process of Dasharo open-source
+firmware update. If your device is currently flashed with the proprietary
+firmware please refer to the [Initial deployment](initial-deployment.md)
+documentation.
+
+The update process may be different, depending on the currently installed
+Dasharo firmware version.
+
+For simplicity of the process we recommend using
+[Dasharo Tools Suite](../../dasharo-tools-suite/overview.md).
+
+Before starting the update procedure be sure to disable Dasharo BIOS Boot
+medium lock and Secure Boot:
+
+1. Power on the device.
+2. While the device is booting, hold the `DELETE` key to enter the UEFI Setup
+    Menu.
+3. Enter the `Dasharo System Features` menu using the arrow keys and Enter.
+4. Enter the [Dasharo Security Options](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+    submenu.
+5. Verify the state of the `Lock the BIOS boot medium` option - if the option
+    is chosen, press `Space` and then `F10` to save the changes.
+6. Go back to the main menu using the `ESC` key.
+7. Enter the `Device Manager` menu.
+8. Enter the [Secure Boot Configuration](https://docs.dasharo.com/dasharo-menu-docs/device-manager/#secure-boot-configuration)
+    submenu.
+9. Verify that the `Current Secure Boot State` field says Disabled - if not,
+    unselect the `Attempt Secure Boot` option below then press `F10` to save
+    the changes.
+10. Reboot the device to properly apply the changes.
+
+The settings of all the above options can be restored after a firmware
+update.
+
+=== "v1000-series"
```

Comment:
I won't repeat the same comments as earlier, please fix accordingly.

Reply:
Firmware update:
https://github.com/Dasharo/docs/compare/9702d18e14289c37406c415ba4b848709fd768cf..7800087bfc37faf292cdb6ef6750eeba07f163de#diff-bf5e0f7b2b116f44501a1ed1ce579b33ca07d492c2391ba6c19e4b112691e950

Initial deployment:
https://github.com/Dasharo/docs/compare/9702d18e14289c37406c415ba4b848709fd768cf..7800087bfc37faf292cdb6ef6750eeba07f163de#diff-c941ed5c4feb42bb6a88fa97784f59b490e632c66d6a3d9d75e4f5a66d046b50

Overview:
https://github.com/Dasharo/docs/compare/9702d18e14289c37406c415ba4b848709fd768cf..7800087bfc37faf292cdb6ef6750eeba07f163de#diff-9d0f901c63656a3f5d2c3d9cdabdfd9123fd0c0db8e7b0b4aaf877b5576e36b4

Post install:
https://github.com/Dasharo/docs/compare/9702d18e14289c37406c415ba4b848709fd768cf..7800087bfc37faf292cdb6ef6750eeba07f163de#diff-d0a11c567d516db7be5b502b4e644e2a768c32815b02d9e1397a5f17ee99645e

Recovery:
https://github.com/Dasharo/docs/compare/9702d18e14289c37406c415ba4b848709fd768cf..7800087bfc37faf292cdb6ef6750eeba07f163de#diff-e41ecb94e6a5ed7028b7ace2692886912e5ac7c09053e5cf5ff6ab3e3315733f

Test matrix:
https://github.com/Dasharo/docs/compare/9702d18e14289c37406c415ba4b848709fd768cf..7800087bfc37faf292cdb6ef6750eeba07f163de#diff-b3a65f84fbcadb548481565d67d078c8e616be9b51f15ee1e07362a94f7e1d52